public/js/tests/tests.js

```diff
@@ -1,90 +1,79 @@
+    ];
 
-  assert.deepEqual(result, [{friendlyName: "BBA"}, {friendlyName: "BBB"}]);
-});
-
-test("should be able to sort an empty list of channels", function(assert) {
-  var channels = [];
+    var result = twiliochat.sortChannelsByName(channels)
 
-  var result = twiliochat.sortChannelsByName(channels)
+    assert.deepEqual(result, [{friendlyName: 'BBA'}, {friendlyName: 'BBB'}]);
+  });
 
-  assert.deepEqual(result, []);
-});
+  it('should be able to sort an empty list of channels', function() {
+    var channels = [];
 
-test("should sort channels when they have same name", function(assert) {
-  var channels = [
-    {friendlyName: "BBB"},
-    {friendlyName: "BBA"},
-    {friendlyName: "BBA"}
-  ];
+    var result = twiliochat.sortChannelsByName(channels)
 
-  var result = twiliochat.sortChannelsByName(channels);
+    assert.deepEqual(result, []);
+  });
 
-  assert.deepEqual(result, [{friendlyName: "BBA"}, {friendlyName: "BBA"}, {friendlyName: "BBB"}]);
-});
+  it('should sort channels when they have same name', function() {
+    var channels = [
+      {friendlyName: 'BBB'},
+      {friendlyName: 'BBA'},
+      {friendlyName: 'BBA'}
+    ];
 
-test("should be able to add messages to chat", function(assert) {
-  var message = {
-    body: "just a test message",
-    author: "me",
-    timestamp: new Date()
-  }
-  var messageList = twiliochat.$messageList;
-  twiliochat.addMessageToList(message);
+    var result = twiliochat.sortChannelsByName(channels);
 
-  assert.ok(messageList.html().indexOf("just a test message") > -1, messageList.html());
-});
+    assert.deepEqual(result, [{friendlyName: 'BBA'}, {friendlyName: 'BBA'}, {friendlyName: 'BBB'}]);
+  });
 
-test("should create a general channel when there is not one", function(){
- var messagingClientMock = { createChannel: function () {} };
- var mock = sinon.mock(messagingClientMock);
- twiliochat.messagingClient = messagingClientMock;
- mock.expects("createChannel").once().returns({ then: function(){} });
- twiliochat.generalChannel = undefined;
- twiliochat.joinGeneralChannel();
+  it('should be able to add messages to chat', function() {
+    var message = {
+      body: 'just a test message',
+      author: 'me',
+      timestamp: new Date()
+    }
+    var messageList = twiliochat.$messageList;
+    twiliochat.addMessageToList(message);
 
- mock.verify();
- ok(true);
-});
+    assert.ok(messageList.html().indexOf('just a test message') > -1, messageList.html());
+  });
 
-test("should not create a new general channel if it already has one", function(){
- var messagingClientMock = { createChannel: function () {} };
- var mock = sinon.mock(messagingClientMock);
- twiliochat.messagingClient = messagingClientMock;
- mock.expects("createChannel").never().returns({then: function(){} });
- twiliochat.generalChannel = {join: function(){ return {then: function() {}}}};
- twiliochat.joinGeneralChannel();
+  it('should create a general channel when there is not one', function(){
```

Comment:
This description is difficult to understand. What about "creates a general channel if not present".